.github/workflows/lerna-publish.yml

```diff
@@ -1,14 +1,22 @@
 name: Publish NPM Packages
 
 on:
+  release:
+    types: published
+
   workflow_dispatch:
     inputs:
       ref:
         description: 'commit/tag/branch to release from'
         required: true
 
 jobs:
-  release:
+  publish:
+    if: >
+      (github.event.inputs) ||
+      (startsWith(github.event.release.tag_name, 'sdk-codegen-all') &&
```

Comment:
using `sdk-codegen-all` tag prefix because it will always be present when a Release PR is merged. It points to the same commit that all the other release tags point to but if we chose `sdk` or `sdk-rtl` etc, any one of those might not show up in a given Release PR if that package had no changes for that release